bin/quickgen

```diff
@@ -185,15 +186,15 @@ maxbin=max(waveMaxbin['b'],waveMaxbin['r'],waveMaxbin['z'])
 
 # Now break the simulated outputs in three different ranges. 
 
-nobj=np.zeros((500,3,maxbin))     # Object Photons
-nsky=np.zeros((500,3,maxbin))      # sky photons
-nivar=np.zeros((500,3,maxbin))     # inverse variance  (object+sky)
-sky_ivar=np.zeros((500,3,maxbin)) # inverse variance of sky
-cframe_observedflux=np.zeros((500,3,maxbin))    # calibrated object flux
-cframe_ivar=np.zeros((500,3,maxbin))    # inverse variance of calibrated object flux
-frame_rand_noise=np.zeros((500,3,maxbin))     # random Gaussian noise to nobj+nsky
-sky_rand_noise=np.zeros((500,3,maxbin))  # random Gaussian noise to sky only
-cframe_rand_noise=np.zeros((500,3,maxbin))  # random Gaussian noise to calibrated flux
+nobj=np.zeros((5000,3,maxbin))     # Object Photons
```

Comment:
I am not sure there will be a single sim file with more than 5000 specs. But if yes then we might want to change that. Earlier I was using 500 for a single spectrograph but now I looped outside of this, so there is 5000 corresponding to 10 spectrographs.